tarxx-example.cpp

```diff
@@ -23,48 +23,191 @@
 // OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-
 #include "tarxx.h"
+#include <filesystem>
+#include <getopt.h>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#if __linux
+#    include <thread>
```

Comment:
I believe you don't need this, please remove

Reply:
removed :)